DeepCat.js

```diff
@@ -49,10 +73,37 @@
 			}
 		} );
 
-		refreshSearchTermMock();
+		if(refreshSearchTermMock())
+			showHint();
 		checkErrorMessage();
 	} );
 
+	function showHint() {
+		if( !mw.cookie.get( makeHintboxCookieToken(mw.msg('hintbox-text'))) ) {
+			var parent= document.getElementById('mw-content-text');
+			var sresults= document.getElementsByClassName('searchresults')[0];
+			var d= parent.insertBefore(document.createElement('div'), sresults);
+			d.style.marginTop= "1em";
+			d.style.marginBottom= "1em";
+			d.innerHTML=
```

Comment:
This code does have a jQuery dependency anyway. Convenience functions like `$.html()` should be used for multiple reasons, starting from compatibility issues (that's what jQuery is for in the first place) to readability because of less function calls and shorter code.